global.json

```diff
@@ -1,7 +1,7 @@
 {
   "sdk": {
-    "version": "6.0.100",
-    "rollForward": "latestFeature"
+    "version": "8.0.0",
+    "rollForward": "latestFeature",
```

Comment:
I did have something to discuss around this.
If `rollForward` is set to `major`, it would allow higher SDK to be loaded, but lower version will still be preferred if present.

I know that Android/iOS workloads are more sensitive to SDK upgrades. I've faced similar problems with CsWinRT that *patch* updates of SDK broke things.
We may choose to pin Android/iOS SDK verions instead.

Reply:
I wasn't intending to change anything here, just to update the version.